sdk/storage/azure-storage-blobs/test/ut/append_blob_client_test.cpp

```diff
@@ -210,53 +210,6 @@ namespace Azure { namespace Storage { namespace Test {
     EXPECT_NO_THROW(appendBlobClient.Delete(options));
   }
 
-  TEST_F(AppendBlobClientTest, SourceBlobAccessConditions)
```

Comment:
Why are we removing this (especially in this PR)? Looks like we might be losing some coverage from `AppendBlobClient`, no?

Reply:
It's moved to `block_blob_client_test.cpp`.

Reply:
Where exactly? I don't see any use of an `AppendBlobClient` or calls to `appendBlobClient.StartCopyFromUri()` within `BlockBlobClientTest`.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/blob/eab04fc80040164203859eeea8840331e53e0c9b/sdk/storage/azure-storage-blobs/test/ut/block_blob_client_test.cpp#L349

Reply:
https://github.com/Azure/azure-sdk-for-cpp/blob/eab04fc80040164203859eeea8840331e53e0c9b/sdk/storage/azure-storage-blobs/test/ut/block_blob_client_test.cpp#L1230

Reply:
Aren't those block blob clients, and not append blob clients?

Reply:
Why does it matter? Asyn copy works for all kinds of blobs.